tests/test_ui_ip_view.py

```diff
@@ -9,79 +9,22 @@
 
 from wtfis.clients.abuseipdb import AbuseIpDbClient
 from wtfis.clients.greynoise import GreynoiseClient
+
+# from wtfis.clients.ip2whois import Ip2WhoisClient  # TODO: Write tests for this
 from wtfis.clients.ipwhois import IpWhoisClient
 from wtfis.clients.shodan import ShodanClient
-from wtfis.clients.urlhaus import UrlHausClient
+
+# from wtfis.clients.urlhaus import UrlHausClient  # TODO: Write tests for this
 from wtfis.models.greynoise import GreynoiseIpMap
 from wtfis.models.ipwhois import IpWhoisMap
-from wtfis.models.passivetotal import Whois as PTWhois
 from wtfis.models.urlhaus import UrlHausMap
 from wtfis.models.virustotal import IpAddress
 from wtfis.models.virustotal import Whois as VTWhois
 from wtfis.ui.view import IpAddressView
 
 
 @pytest.fixture()
-def view01(
-    test_data,
-    mock_abuseipdb_get,
-    mock_ipwhois_get,
-    mock_shodan_get_ip,
-    mock_greynoise_get,
-    mock_urlhaus_get,
-):
-    """1.1.1.1 with PT whois. Complete test of all panels. Also test print()."""
-    ip = "1.1.1.1"
-
-    geoasn_pool = json.loads(test_data("ipwhois_1.1.1.1.json"))
-    geoasn_client = IpWhoisClient()
-    geoasn_client._get_ipwhois = MagicMock(
-        side_effect=lambda ip: mock_ipwhois_get(ip, geoasn_pool)
-    )
-    geoasn_enrich = geoasn_client.enrich_ips(ip)
-
-    shodan_pool = json.loads(test_data("shodan_1.1.1.1.json"))
-    shodan_client = ShodanClient(MagicMock())
-    shodan_client._get_ip = MagicMock(
-        side_effect=lambda ip: mock_shodan_get_ip(ip, shodan_pool)
-    )
-    shodan_enrich = shodan_client.enrich_ips(ip)
-
-    greynoise_pool = json.loads(test_data("greynoise_1.1.1.1.json"))
-    greynoise_client = GreynoiseClient("dummykey")
-    greynoise_client._get_ip = MagicMock(
-        side_effect=lambda ip: mock_greynoise_get(ip, greynoise_pool)
-    )
-    greynoise_enrich = greynoise_client.enrich_ips(ip)
-
-    abuseipdb_pool = json.loads(test_data("abuseipdb_1.1.1.1_red.json"))
-    abuseipdb_client = AbuseIpDbClient("dummykey")
-    abuseipdb_client._get_ip = MagicMock(
-        side_effect=lambda ip: mock_abuseipdb_get(ip, abuseipdb_pool)
-    )
-    abuseipdb_enrich = abuseipdb_client.enrich_ips(ip)
-
-    urlhaus_pool = json.loads(test_data("urlhaus_1.1.1.1.json"))
-    urlhaus_client = UrlHausClient()
-    urlhaus_client._get_host = MagicMock(
-        side_effect=lambda ip: mock_urlhaus_get(ip, urlhaus_pool)
-    )
-    urlhaus_enrich = urlhaus_client.enrich_ips(ip)
-
-    return IpAddressView(
-        console=Console(),
-        entity=IpAddress.model_validate(json.loads(test_data("vt_ip_1.1.1.1.json"))),
-        geoasn=geoasn_enrich,
-        whois=PTWhois.model_validate(json.loads(test_data("pt_whois_1.1.1.1.json"))),
-        shodan=shodan_enrich,
-        greynoise=greynoise_enrich,
-        abuseipdb=abuseipdb_enrich,
-        urlhaus=urlhaus_enrich,
-    )
```

Comment:
As before, we still need this fixture, and VTWhois can be used here as replacement.